src/ModelHelpers/Relations.php

```diff
@@ -206,4 +208,54 @@ protected function newHasMany($query, $parent, $foreignKey, $localKey)
 	{
 		return new HasMany($query, $parent, $foreignKey, $localKey);
 	}
+
+	/**
+	 * Define an inverse one-to-one or many relation.
+	 *
+	 * @param   string  $related    related Model
+	 * @param   string  $foreignKey foreign key name in current mode
+	 * @param   string  $ownerKey   the associated key on the parent model.
+	 * @return \Joomla\Entity\Relations\BelongsTo
+	 */
+	public function belongsTo($related, $foreignKey = null, $ownerKey = null)
+	{
+		$relation = $this->getRelationFromCallerFunction();
+
+		$instance = $this->newRelatedInstance($related);
+
+		$foreignKey = $foreignKey ?: Normalise::toUnderscoreSeparated($relation) . '_' . $instance->getPrimaryKey();
+
+		$ownerKey = $ownerKey ?: $instance->getPrimaryKey();
+
+		return $this->newBelongsTo(
+			$instance->newQuery(), $this, $foreignKey, $ownerKey, $relation
+		);
+	}
+
+	/**
+	 * Instantiate a new BelongsTo relation.
+	 *
+	 * @param   Query   $query      Query instance
+	 * @param   Model   $child      child Model instance
+	 * @param   string  $foreignKey foreign key name
+	 * @param   string  $ownerKey   the associated key on the parent model.
+	 * @param   string  $relation   relation name
+	 * @return \Joomla\Entity\Relations\BelongsTo
+	 */
+	protected function newBelongsTo(Query $query, Model $child, $foreignKey, $ownerKey, $relation)
+	{
+		return new BelongsTo($query, $child, $foreignKey, $ownerKey, $relation);
+	}
+
+	/**
+	 * Get the relation name from the caller function
+	 *
+	 * @return string
+	 */
+	protected function getRelationFromCallerFunction()
```

Comment:
Yeah let's not do this. I'd much rather hardcode a string that do this voodoo :)